pkg/eval/mods/file/file_test.go

```diff
@@ -0,0 +1,29 @@
+package file
+
+import (
+	"testing"
+
+	"src.elv.sh/pkg/eval"
+	"src.elv.sh/pkg/eval/errs"
+	"src.elv.sh/pkg/eval/evaltest"
```

Comment:
Follow the convention in other module implementations and use a dot import (see https://golang.org/ref/spec#Import_declarations) to import `evaltest`.